CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing
```

Comment:
Can we also add a line which tells the contributors about bug deadline?

Reply:
The 60 day bug TAT is an internal goal and our contributors are not responsible for meeting that metric. If a bug that an external contributor is working starts nearing the 60 day deadline, we will have to pick up the remaining work to get the bug wrapped up.